filters/builtin/static.go

```diff
@@ -35,11 +37,13 @@ func (spec *static) CreateFilter(config []interface{}) (filters.Filter, error) {
 
 	webRoot, ok := config[0].(string)
 	if !ok {
+		log.Warn("Invalid parameter type for web root prefix. Expected string, got %s", reflect.TypeOf(config[0]))
```

Comment:
please change %s to %v because %s is for strings, while reflect.TypeOf(interface{}) returns a reflect.Type and for generic values fmt expects %v